src/store/traction/pacbio/plates/actions.js

```diff
@@ -1,12 +1,16 @@
-import handleResponse from '@/api/v1/ResponseHelper'
+import handleResponse from '@/api/v2/ResponseHelper'
 import { groupIncludedByResource } from '@/api/JsonApi'
 
-const setPlates = async ({ commit, getters }, filter, page) => {
+const setPlates = async ({ commit, getters }, options) => {
```

Comment:
Does this make things better? It feels less explicit. Did we not create a type to pass into requests?

Reply:
`setPlates` is called using composable function `fetchWithQueryParams`  and it should be an object because that's how fetchWithQueryParams is wrapping parameters which is as follows

` const page = { size: page_size.value.toString(), number: page_number.value.toString() }
    const filter = buildFilter(filterOptions)
    const { success, errors, meta } = await fetcher({ page, filter })`

I didn't want to change the composable function since it might be used in many places. 

The parameter name changed to 'filterOptions' to make it more clear.